pkg/autostart/io.lima-vm.autostart.INSTANCE.plist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Label</key>
+	<string>io.lima-vm.{{ .INSTANCE }}</string>
+	<key>ProgramArguments</key>
+	<array>
+		<string>'"$(which limactl)"'</string>
```

Comment:
Does `$()` really work here?

Reply:
I don't think so. Everything should be filled in via Go templating; there should be no shell interpolation going on.